packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -166,6 +166,9 @@ export const Datepicker: ComponentWithAs<'div', DatepickerProps> &
   setStart();
   const inputRef = React.useRef<HTMLElement>();
 
+  // FIXME: This object is created every render, causing a cascade of useCallback/useEffect re-runs.
```

Comment:
This is a potentially significant performance problem where a proper fix wasn't obvious to me without more detailed knowledge of the desired component behavior. @pompomon FYI, might be worth coming back with a fix later.